assignments/callbacks.js

```diff
@@ -26,26 +26,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  return cb(arr.length)
 }
 
+getLength(items, function(length){
+  console.log(length);
+})
+
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  let lastItem = arr.pop();
+  arr.push(lastItem);
+  return cb(lastItem)
 }
 
+last(items, function(last){
+  console.log(last)
+})
+
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y)
 }
 
+sumNums(15, 3, function(sum){
+  console.log(sum)
+})
+
 function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y)
 }
 
+multiplyNums(10, 3, function(product){
+  console.log(product)
+})
+
 function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
+  for (let i = 0; list.length; i++){
+    if (item == list[i]){
+      return cb(true)
+    } 
+  }
 }
 
+contains('Gum', items, function(bool){
+  console.log(bool)
+})
+
```

Comment:
so this totally works, only thing I'd say for the sake of keeping it DRY, you could do something like this `const logger = value => console.log(value)` and use it in place of the cb's that you're re-writing everywhere